lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -61,6 +61,55 @@ def register_get_thread_response(self, thread):
             status=200
         )
 
+    def register_post_comment_response(self, response_overrides, thread_id=None, parent_id=None):
+        """
+        Register a mock response for POST on the CS comments endpoint for the
+        given thread or parent; exactly one of thread_id and parent_id must be
+        specified.
+        """
```

Comment:
I think you mentioned that there will be a clean up after all this is done. I think adding parameter docs would be helpful in that clean up even though these are just mocks.